vigenere-cipher-main/src/cypher.py

```diff
@@ -0,0 +1,23 @@
+#vigenere cipher
+def vigenere_cipher(text, key_stream, encrypting):
+    result = ""
+    for i in range(len(text)):
+        char = text[i]
+        if encrypting:
+            result += chr((ord(char) - 97 + ord(key_stream[i]) - 97) % 26 + 97)
+        else:
+            result += chr((ord(char) - ord(key_stream[i]) + 26) % 26 + 97)

```

Comment:
What are these operations doing? This should be documented, either by splitting the operations up and assigning variables, or describing why the operations are done the way they are.